tests/test_commands.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
 # Verify all options are plumbed through to the MT.
-# TODO Replace "stevemessick" with "$USER" so others can run this script.
+# Set envar KAGGLE_DEVELOPER to the Kaggle user name (probably already done).
```

Comment:
is this a `TODO`?  Or should the script check if it's already set and prompt the user to set it?

Reply:
This script is a WIP. Lots of work should be done to make it generally usable. I don't know if it has enough long-term value to finish; I may delete it. It's purpose is to give me confidence to move out of beta, which it has done.